docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -0,0 +1,829 @@
+---
+title: SEP-24 User Guide
+sidebar_position: 10
+---
+
+import { CodeExample } from "@site/src/components/CodeExample";
+
+This guide will walk you through installing, configuring, and integrating with the Anchor Platform for the purpose of building an on & off-ramp service compatible with [SEP-24][sep-24], the ecosystem's standardized protocol for hosted deposits and withdrawals.
+
+By leveraging the Anchor Platform's support for SEP-24, businesses make their on & off-ramp service available as an in-app experience through Stellar-based applications such as wallets and exchanges, extending their reach and connecting with users through the applications they already use.
+
+:::note
+
+As we improve the documentation, parts of this guide that are relevant to other use cases may be moved into their own sections.
+
+:::
+
+## Installation
+
+The easiest way to install the Anchor Platform is to pull the [docker image][anchor-platform-image]. SEP-24 support is not yet officially released, but is available using the `edge` tag.
+
+<CodeExample>
+
+```bash
+docker pull stellar/anchor-platform:edge
+```
+
+</CodeExample>
+
+## Configuration
+
+In this guide we'll use [docker compose][docker-compose] for simplicity, but you can run the Anchor Platform using other tools that support docker as well, such as [minikube] or a full-blown [kubernetes] cluster.
+
+Let's create a minimal compose file to get started.
+
+<CodeExample>
+
+```yaml
+# docker-compose.yml
+version: "3.8"
+services:
+platform-server:
+  image: stellar/anchor-platform:edge
+  command: --sep-server
+  ports:
+    - "8080:8080"
+  env_file:
+    - ./config/dev.env
+  volumes:
+    - ./config:/home
+```
+
+</CodeExample>
+
+The `--sep-server` option tells the Anchor Platform to make the API endpoints defined by the SEPs you've enabled via configuration on port 8080. This option also makes the Platform API available, which is the backend API your service(s) will use to communicate with the Anchor Platform.
+
+:::note
+
+A future release of the Anchor Platform will add an additional command line option for the Platform API, and update the `--sep-server` option to only make the SEP APIs available.
+
+:::
+
+### Create a Configuration File
+
+The Anchor Platform supports two approaches for configuration: using environment variables or a YAML configuration file. While it's possible to only use one of the two approaches, most deployments will use both due to the a variables value structure or security requirements.
```

Comment:
"...due to the a variables value..." there seems to be an extra word here, but I'm not sure which. Should it be "...due to the variable's value..."?